cdn/dev/keyboard-search/install.js

```diff
@@ -61,6 +61,34 @@ function startAfterPageLoad_Windows(data) {
           location.href = downloadUrl;
       }, 1000);
 
+      window.addEventListener('blur', function() {
+        window.clearTimeout(fallbackHandle);
+       });
+    }
+  }, 10);
+}
+
+/**
+ * Redirects to keyman:// to trigger Keyman Configuration, and if that fails,
+ * to the standard download url for the package to install. This currently only
+ * applies to Linux users.
+ * @param {object} data from the object with properties for host, tier, version, id, bcp47, name
+ */
+function startAfterPageLoad_Linux(data) {
```

Comment:
Given this is near identical to the Windows code above it, I think I'd like to see the two functions merged... I know it's a little more hassle but it will be less maintenance in the future :smile:

Reply:
Done in a separate PR.

Reply:
I am the very model of a consistent somethingorother.... Sorry!